app/controllers/aspects_controller.rb

```diff
@@ -14,10 +14,15 @@ def create
     aspecting_person_id = params[:person_id]
 
     if @aspect.save
+      result = {id: @aspect.id, name: @aspect.name}
       if aspecting_person_id.present?
         connect_person_to_aspect(aspecting_person_id)
+        if @contact
+          result[:aspect_membership] = AspectMembershipPresenter.new(@contact.aspect_memberships.last).base_hash
```

Comment:
`.last`/`first` without an explicit `order` is not deterministic iirc. Is this supposed to return the one created in line 19? Perhaps that should return it?